addons/account/models/account_journal_dashboard.py

```diff
@@ -61,13 +61,16 @@ def _get_json_activity_data(self):
         today = fields.Date.context_today(self)
         activities = defaultdict(list)
         # search activity on move on the journal
-        sql_query = """
+        lang = self.env.user.lang or get_lang(self.env).code
+        act_name = f"COALESCE(act_type.name->>'{lang}', act_type.name->>'en_US')" \
```

Comment:
question here: what about SQL injection? wouldn't it be better to pass lang in the cr.execute params?